hashes/zkevm-keccak/src/keccak_packed_multi.rs

```diff
@@ -1559,21 +1548,25 @@ impl<F: Field> KeccakCircuitConfig<F> {
         ] {
             assign_fixed_custom(region, *column, offset, *value);
         }
-
-        assign_advice_custom(
-            region,
-            self.keccak_table.is_enabled,
-            offset,
-            Value::known(F::from(row.is_final)),
-        );
-
+        assign_advice_custom(region, self.is_final, offset, Value::known(F::from(row.is_final)));
+        let mut result = Vec::new();
         // Cell values
-        row.cell_values.iter().zip(self.cell_manager.columns()).for_each(|(bit, column)| {
-            assign_advice_custom(region, column.advice, offset, Value::known(*bit));
-        });
-
+        row.cell_values.iter().zip(self.cell_manager.columns()).enumerate().for_each(
+            |(col_index, (bit, column))| {
+                let assigned_advice =
+                    assign_advice_custom(region, column.advice, offset, Value::known(*bit));
+                if flags.contains(&col_index) {
+                    result.push(assigned_advice);
+                }
+            },
+        );
         // Round constant
         assign_fixed_custom(region, self.round_cst, offset, row.round_cst);
+        result
+    }
+
+    pub fn set_row(&self, region: &mut Region<'_, F>, offset: usize, row: &KeccakRow<F>) {
+        self.set_row_with_flags(region, offset, row, &[]);
```

Comment:
`set_row_with_flags` returns a `Vec<KeccakAssignedValue<F>>`, why don't we use it here?

Reply:
Or maybe my question is why `set_row_with_flags` returns it?